lib/skins.js

```diff
@@ -6,6 +6,7 @@
 
 'use strict';
 var SKIN_FILES_PATTERN = '**/*(*.css|*.scss|*.sass|*.ftl|*.png|*.svg|*.jpg|*.jpeg|skin.properties)';
+const sp = require('synchronized-promise')
```

Comment:
Is **synchronized-promise** already in package.json?

Reply:
yes, i added this lib to package.json as part of this PR - https://github.com/lithiumtech/lithium-sdk/pull/141